packages/tools/client-debugger/client-debugger-view/src/components/TreeView.tsx

```diff
@@ -5,27 +5,21 @@
 
 import React from "react";
 import { HasContainerId, VisualTreeNode } from "@fluid-tools/client-debugger";
-import { Accordion } from "./utility-components/";
-import { TreeDataView } from "./TreeDataView";
+import { TreeRenderHelper } from "./TreeRenderHelper";
 
 /**
  * {@link TreeView} input props.
  */
 export interface TreeViewProps extends HasContainerId {
+	nodeKey: string | undefined;
```

Comment:
Nit: I might call this something like "label", since in terms of our components / visualizations, it's just a label for the corresponding node. But this is just a Josh opinion :)

Reply:
Changed to `label`. Always a huge fan of Josh opinion™️ 🫡